spec/execution_tests.lua

```diff
@@ -1425,6 +1425,166 @@ function execution_tests.run(compile_file, backend, _ENV, only_compile)
         end)
     end)
 
+    -- Note: Implementation currenly only supports float and uses fabs().
+    -- Test avoids differences that may be caused by needing integers.
```

Comment:
Maybe it would be nice if we said in the manual.md that the math builtins only accept floating point numbers. And also mention in the manual.md that the math builtins exist :)

Reply:
This reminds me that for every update to the upstream repo, we'll have to change docs accordingly as well.

Reply:
That certainly is a downside of keeping things in a separate repository. But while we're still experimenting with restructuring the docs, I think there are advangages to keeping it separate. Once we stabilize, we can consider if we want to move the md files back to the main repo, or if we want to continue keeping them separate.

Reply:
I made the changes you requested and just pushed them. The elephant in the room is still the translator test mismatch failure.

Reply:
The pull-request comments might not be the best place to discuss this. This comment thread is attached to a line of code and if that line of code gets edited, then the commends under it may end up hidden. I'm going to continue the discussion back on #490.